src/omnibelt/map-parallel-limit-p.js

```diff
@@ -1,19 +1,36 @@
 const curry = require('ramda/src/curry');
 const times = require('ramda/src/times');
+const allSettledP = require('./all-settled-p');
 
 const objAccumulator = (accumulator, transformed, original) => { accumulator[original[0]] = transformed; };
 const aryAccumulator = (accumulator, transformed) => { accumulator.push(transformed); };
 
 const makeEvaluator = (func, iterator, accFunc, accumulator) => {
   const evaluator = async () => {
     const next = iterator.next();
-    if (next.done) { return; }
+    if (next.done) { return 'done'; }
     accFunc(accumulator, await func(next.value), next.value);
-    await evaluator();
   };
   return evaluator;
 };
 
+const runner = async (maxParallel, evaluator) => {
```

Comment:
this can be redone in a similar way to the way you changes mapP / resolveProps - in fact you don't even need a fancy runner function, I don't think